README.md

```diff
@@ -73,6 +73,21 @@ data. Make sure that the Redis server you use has persistence enabled - your
 redis.conf should have contents similar to the section `**PERSISTENCE**` of
 redis.conf in the example config files linked in http://redis.io/topics/config
 
+# Contributing #
+You're encouraged to contribute to the uniqush-push project. There are two ways you can contribute.
+
+# Issues #
```

Comment:
I think you use two (`##`) on each side for subheadings.

http://maruku.rubyforge.org/maruku.md http://maruku.rubyforge.org/maruku.html